docs/operation-guides/50-kiali-grafana.md

```diff
@@ -1,61 +1,52 @@
 ---
 title: Access and Expose Kiali and Grafana
-type: Configuration
 ---
 
-By default, Kyma does not expose Kiali and Grafana. However, you can still access them using port forwarding. Also, read [Expose Kyma UIs securely](http://tbd) to learn how to expose Kiali and Grafana securely using an identity provider of your choice.
+By default, Kyma does not expose Kiali and Grafana. However, you can still access them using port forwarding. If you want to expose Kiali and Grafana securely, use an identity provider of your choice.
 
-### Prerequisites
+## Prerequisites
 
 - You have defined the kubeconfig file for your cluster as default (see [Kubernetes: Organizing Cluster Access Using kubeconfig Files](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/)).
+- To expose the services securely with OAuth, you have a registered OAuth application with one of the [supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider#github-auth-provider).
```

Comment:
```suggestion
- To expose the services securely with OAuth, you must have a registered OAuth application with one of the [supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider#github-auth-provider).
```

Reply:
Do we use "must" in prerequisites? We never used it in my previous projects, but I'm fine either way :)

Reply:
I corrected it as this sentence sounded somehow incomplete. You can use "make sure" or sth like that instead. But "must" is ok, I guess ;)